karma.conf.js

```diff
@@ -0,0 +1,62 @@
+// Karma configuration
+// Generated on Tue Dec 08 2015 23:01:01 GMT-0800 (Pacific Standard Time)
+
+module.exports = function (config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['browserify', 'jasmine'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'spec/helpers/marble-testing.js',
+      'spec/helpers/test-helper.js',
+      'spec/**/*-spec.js'
+    ],
+
+    // list of files to exclude
+    exclude: [
+      'spec/observables/from-promise-spec.js' //'should globally throw unhandled errors' fails
+    ],
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      'spec/**/*.js': ['browserify']
+    },
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
+
+    // web server port
+    port: 9876,
+
+    // enable / disable colors in the output (reporters and logs)
+    colors: true,
+
+    // level of logging
+    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+    logLevel: config.LOG_INFO,
+
+    // enable / disable watching file and executing tests whenever any file changes
+    autoWatch: false,
+
+    // start these browsers
+    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
+    browsers: ['Chrome'],
+
+    // Continuous Integration mode
+    // if true, Karma captures browsers, runs the tests and exits
+    singleRun: true,
+
+    // Concurrency level
+    // how many browser should be started simultanous
+    concurrency: Infinity
```

Comment:
I would default to `concurrency: 1`, because I have some experience seeing browsers failing their tests just because they are not the currently focused window, and we can afford waiting a bit longer on CI for all tests to complete.